pyproject.toml

```diff
@@ -1,35 +1,38 @@
-[tool.poetry]
+[project]
 name = "blurring_as_a_service"
-version = "2.0.0"
+version = "2.1.0"
 description = ""
 authors = [
-    "Sebastian Davrieux <s.davrieux@amsterdam.nl>",
-    "Andrea Lombardo <a.lombardo@amsterdam.nl>",
-    "Diana Epureanu <d.epureanu@amsterdam.nl>",
-    "Chris Eijgenstein",
-    "Daan Bloembergen <d.bloembergen@amsterdam.nl>"
+    {name = "Sebastian Davrieux", email = "s.davrieux@amsterdam.nl"},
+    {name = "Diana Epureanu", email = "d.epureanu@amsterdam.nl"},
+    {name = "Andrea Lombardo", email = "a.lombardo@amsterdam.nl"},
+    {name = "Daan Bloembergen", email = "d.bloembergen@amsterdam.nl"},
+    {name = "Chris Eijgenstein", email = ""},
+]
+maintainers = [
+    { name = "Computer Vision Team, Gemeente Amsterdam", email = "cvt@amsterdam.nl" },
+]
+requires-python = ">=3.8, <=3.12"
```

Comment:
This fails when I try to run things, should be `>=3.10, <=3.12` since otherwise it conflicts with YoloDevKit dependency and UV complains.